src/Integration.Vsix/ErrorList/SonarErrorListDataSource.cs

```diff
@@ -186,7 +187,23 @@ public IEnumerable<IAnalysisIssueLocationVisualization> GetLocations(string file
 
         public void LocationsUpdated(IEnumerable<string> affectedFilePaths)
         {
-            // TODO
+            if (affectedFilePaths == null)
```

Comment:
Will be called by the tagger when it has recalculated and updated the spans.
The data source will increment the snapshot version number then notify the error list that the data has changed so it can update the line/position if necessary.